aws_streams/streams_single_region.yaml

```diff
@@ -24,41 +24,67 @@ Parameters:
     Type: String
-    Default: ''
+    Default: ""
   SecondNamespace:
     Description: >-
       A namespace to use for filtering. Leave blank if you do not need to filter by namespace.
     Type: String
-    Default: ''
+    Default: ""
   ThirdNamespace:
     Description: >-
       A namespace to use for filtering. Leave blank if you do not need to filter by namespace.
       If you need to filter more than 3 namespaces, manually edit the settings for the streams
       within the AWS CloudWatch Console after the stack is created successfully.
     Type: String
-    Default: ''
+    Default: ""
   DdSite:
     Description: >-
       Define your Datadog Site to send data to. For the Datadog EU site, set to datadoghq.eu
     Type: String
     AllowedPattern: .+
-    Default: 'datadoghq.com'
+    Default: "datadoghq.com"
     ConstraintDescription: DdSite is required
 Conditions:
-  HasIncludeNamespace1: !And [ !Not [ !Equals [ !Ref FirstNamespace, '' ] ], !Equals [ !Ref FilterMethod, 'Include' ]]
-  HasIncludeNamespace2: !And [ !Not [ !Equals [ !Ref SecondNamespace, '' ] ], !Equals [ !Ref FilterMethod, 'Include' ]]
-  HasIncludeNamespace3: !And [ !Not [ !Equals [ !Ref ThirdNamespace, '' ] ], !Equals [ !Ref FilterMethod, 'Include' ]]
-  HasExcludeNamespace1: !And [ !Not [ !Equals [ !Ref FirstNamespace, '' ] ], !Not [ !Equals [ !Ref FilterMethod, 'Include' ]]]
-  HasExcludeNamespace2: !And [ !Not [ !Equals [ !Ref SecondNamespace, '' ] ], !Not [ !Equals [ !Ref FilterMethod, 'Include' ]]]
-  HasExcludeNamespace3: !And [ !Not [ !Equals [ !Ref ThirdNamespace, '' ] ], !Not [ !Equals [ !Ref FilterMethod, 'Include' ]]]
-  EUDatacenter: !Equals [ !Ref DdSite, 'datadoghq.eu' ]
-  US5Datacenter: !Equals [ !Ref DdSite, 'us5.datadoghq.com' ]
-  Staging: !Equals [ !Ref DdSite, 'datad0g.com' ]
+  IsInclude: !Equals [!Ref FilterMethod, "Include"]
+  IsExclude: !Not [!Equals [!Ref FilterMethod, "Include"]]
+  HasIncludeNamespace1:
+    !And [
+      !Not [!Equals [!Ref FirstNamespace, ""]],
+      !Equals [!Ref FilterMethod, "Include"],
+    ]
+  HasIncludeNamespace2:
+    !And [
+      !Not [!Equals [!Ref SecondNamespace, ""]],
+      !Equals [!Ref FilterMethod, "Include"],
+    ]
+  HasIncludeNamespace3:
+    !And [
+      !Not [!Equals [!Ref ThirdNamespace, ""]],
+      !Equals [!Ref FilterMethod, "Include"],
+    ]
+  HasExcludeNamespace1:
+    !And [
+      !Not [!Equals [!Ref FirstNamespace, ""]],
+      !Not [!Equals [!Ref FilterMethod, "Include"]],
+    ]
+  HasExcludeNamespace2:
+    !And [
+      !Not [!Equals [!Ref SecondNamespace, ""]],
+      !Not [!Equals [!Ref FilterMethod, "Include"]],
+    ]
+  HasExcludeNamespace3:
+    !And [
+      !Not [!Equals [!Ref ThirdNamespace, ""]],
+      !Not [!Equals [!Ref FilterMethod, "Include"]],
+    ]
+  EUDatacenter: !Equals [!Ref DdSite, "datadoghq.eu"]
+  US5Datacenter: !Equals [!Ref DdSite, "us5.datadoghq.com"]
+  Staging: !Equals [!Ref DdSite, "datad0g.com"]
```

Comment:
Does this template work for AP1? Do we need a ticket for adding AP1 support?

Reply:
Yeah, we should add support for ap1 -- I'll make a follow up ticket for this